opslevel/resource_opslevel_service_tag.go

```diff
@@ -2,13 +2,18 @@ package opslevel
 
 import (
 	"fmt"
+	"regexp"
 	"strings"
 
 	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
 	"github.com/opslevel/opslevel-go"
 )
 
+// Tag key names are stroed in OpsLevel as lowercase so we need to ensure the configuration is written as lowercase
+var TagKeyRegex = regexp.MustCompile(`\A[a-z][0-9a-z_\.\/\\-]*\z`)
```

Comment:
Are the backticks so that you don't have to escape anything and/or so go doesn't interpret any special characters? Just curious.

Reply:
yes!  vscode go tools told me to do that ;)

Reply:
Awesome! 🎉